task_sdk/src/airflow/sdk/definitions/_internal/abstractoperator.py

```diff
@@ -136,6 +138,39 @@ class AbstractOperator(Templater, DAGNode):
         )
     )
 
+    @cached_property
+    def operator_extra_link_dict(self) -> dict[str, Any]:
+        """Returns dictionary of all extra links for the operator."""
+        op_extra_links_from_plugin: dict[str, Any] = {}
+        from airflow import plugins_manager
+
+        plugins_manager.initialize_extra_operators_links_plugins()
+        if plugins_manager.operator_extra_links is None:
+            raise AirflowException("Can't load operators")
+        for ope in plugins_manager.operator_extra_links:
+            if ope.operators and self.operator_class in ope.operators:
+                op_extra_links_from_plugin.update({ope.name: ope})
```

Comment:
I still think this should be handled in deserialization, not at task run time, but very open to discussion on this.

WDYT?

Reply:
We should yes, that will reduce the unwanted dependency between plugins manager and the task sdk. We should keep it a minimal slim package anyways.

Its not just global_operator_extra_link_dict, there's also operator_extra_link_dict and extra_links which I will move to their right places in the next commit